diff -Nru quakeforge-0.2.99beta6/configure.in quakeforge-0.2.99beta6.new/configure.in
--- quakeforge-0.2.99beta6/configure.in	Mon Jan 29 13:52:11 2001
+++ quakeforge-0.2.99beta6.new/configure.in	Mon Jan 29 13:56:07 2001
@@ -950,79 +950,6 @@
 fi
 AC_DEFINE_UNQUOTED(FS_USERPATH, "$userpath")
 
-dnl CFLAGS for release and devel versions
-CFLAGS=""
-AC_ARG_ENABLE(debug,
-	[  --enable-debug          compile with debugging (for development)],
-	debug=$enable_debug
-)
-AC_MSG_CHECKING(for debugging)
-if test "x$debug" = xyes; then
-	AC_MSG_RESULT(yes)
-	BUILD_TYPE="$BUILD_TYPE Debug"
-	CFLAGS="$CFLAGS -g"
-else
-	AC_MSG_RESULT(no)
-fi
-
-AC_ARG_ENABLE(optimize,
-	[  --disable-optimize      compile without optimizations (for development)],
-	optimize=$disable_optimize,
-	optimize=yes
-)
-AC_ARG_WITH(amd,
-	[  --with-amd              Optimize for AMD processors instead of Intel],
-	HAVE_AMD="yes",
-	HAVE_AMD="no"
-)
-AC_MSG_CHECKING(for optimization)
-if test "x$optimize" = xyes; then
-	AC_MSG_RESULT(yes)
-	BUILD_TYPE="$BUILD_TYPE Optimize"
-	if test "x$GCC" = xyes; then
-		CFLAGS="$CFLAGS -O6 -ffast-math -funroll-loops -fomit-frame-pointer -fexpensive-optimizations"
-		AC_MSG_CHECKING(for special compiler settings)
-		case "${host_cpu}" in
-			i?86)
-				if test "x$HAVE_AMD" = "xyes"; then
-					MORE_CFLAGS="-march=k6 -malign-loops=2 -malign-jumps=2 -malign-functions=2"
-				else
-					MORE_CFLAGS="-march=${host_cpu} -malign-loops=2 -malign-jumps=2 -malign-functions=2"
-				fi
-				;;
-			*)
-				MORE_CFLAGS=""
-				;;
-		esac
-		if test "x$MORE_CFLAGS" = x; then
-			AC_MSG_RESULT(no)
-		else
-			AC_MSG_RESULT(yes)
-			CFLAGS="$CFLAGS $MORE_CFLAGS"
-		fi
-	else
-		CFLAGS=-O2
-	fi
-else
-	AC_MSG_RESULT(no)
-	CFLAGS="$CFLAGS -O"
-fi
-
-dnl CFLAGS for release and devel versions
-AC_ARG_ENABLE(profile,
-	[  --enable-profile        compile with profiling (for development)],
-	profile=$enable_profile
-)
-if test "x$profile" = xyes; then
-	BUILD_TYPE="$BUILD_TYPE Profile"
-	if test "x$GCC" = xyes; then
-		CFLAGS="`echo $CFLAGS | sed -e 's/-fomit-frame-pointer//g'` -pg"
-		LDFLAGS="$LDFLAGS -pg"
-	else
-		CFLAGS="$CFLAGS -p"
-	fi
-fi
-
 check_pipe=no
 if test "x$GCC" = xyes; then
 	dnl Check for -pipe vs -save-temps.
