--- quakeforge-0.5.4/configure.ac.orig	2003-07-18 00:35:48.000000000 +0200
+++ quakeforge-0.5.4/configure.ac	2004-04-11 12:21:22.816309896 +0200
@@ -816,7 +816,7 @@
 				[
 #include <sys/asoundlib.h>
 #if defined(SND_LIB_MAJOR) && defined(SND_LIB_MINOR)
-#if SND_LIB_MAJOR > 0 || (SND_LIB_MAJOR == 0 && SND_LIB_MINOR == 5)
+#if (SND_LIB_MAJOR == 0 && SND_LIB_MINOR == 5)
 QF_maGiC_VALUE
 #endif
 #endif
