--- quakeforge-0.5.4/ruamoko/lib/Makefile.am.orig	2003-04-13 22:43:52.000000000 +0200
+++ quakeforge-0.5.4/ruamoko/lib/Makefile.am	2004-04-11 19:17:22.136009670 +0200
@@ -1,9 +1,9 @@
 AUTOMAKE_OPTIONS=	foreign
 
-libdir=$(prefix)/lib/ruamoko
+ruamokolibdir=$(libdir)/ruamoko
 
 QFCC=$(top_builddir)/tools/qfcc/source/qfcc$(EXEEXT)
-QCFLAGS=-qq -g -Werror
+QCFLAGS=-qq -Werror
 QCPPFLAGS=$(INCLUDES)
 PAK=$(top_builddir)/tools/pak/pak$(EXEEXT)
 RANLIB=touch
@@ -17,7 +17,7 @@
 libs=
 endif
 
-lib_LIBRARIES= $(libs)
+ruamokolib_LIBRARIES= $(libs)
 EXTRA_LIBRARIES= $(ruamoko_libs)
 
 %.qfo: %.r
