diff -uNr quakeforge-0.3.0.old/include/compat.h quakeforge-0.3.0/include/compat.h
--- quakeforge-0.3.0.old/include/compat.h	Thu Feb  8 10:17:29 2001
+++ quakeforge-0.3.0/include/compat.h	Mon Aug 12 11:16:13 2002
@@ -87,5 +87,8 @@
 #if !defined(strncaseequal)
 # define strncaseequal(a,b,c) (strncasecmp (a, b, c) == 0)
 #endif
+#if !defined(stricmp)
+# define stricmp strcasecmp
+#endif
 
 #endif // _COMPAT_H
